test_scripts/Smoke/Resumption/ATF_Resumption_unexpected_disconnect.lua

```diff
@@ -0,0 +1,174 @@
+
+function Test:StartSDL_With_One_Activated_App()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+          self:startSession():Do(function ()
+            commonFunctions:userPrint(35, "App is registered")
+            default_app = self.applications[default_app_params.appName]
+            commonSteps:ActivateAppInSpecificLevel(self, default_app)
+            EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL",audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+            commonFunctions:userPrint(35, "App is activated")
+          end)
+        end)
+      end)
+    end)
+  end)
+end
+
+function Test:AddCommand()
+  local cid = self.mobileSession:SendRPC("AddCommand", { cmdID = 1, vrCommands = {"OnlyVRCommand"}})
+  local on_hmi_call = EXPECT_HMICALL("VR.AddCommand", {cmdID = 1, type = "Command",
+                      vrCommands = {"OnlyVRCommand"}})
+  on_hmi_call:Do(function(_, data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+  EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+    self.currentHashID = data.payload.hashID
+  end)
+end
+
+function Test:AddSubMenu()
+  local cid = self.mobileSession:SendRPC("AddSubMenu", { menuID = 1, position = 500,
+              menuName = "SubMenu"})
+  local on_hmi_call = EXPECT_HMICALL("UI.AddSubMenu", { menuID = 1, menuParams = 
+                              { position = 500, menuName = "SubMenu"}})
+  on_hmi_call:Do(function(_, data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+  EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+    self.currentHashID = data.payload.hashID
+  end)
+end
+
+function Test:AddChoiceSet()
+  local cid = self.mobileSession:SendRPC("CreateInteractionChoiceSet", {interactionChoiceSetID = 1,
+      choiceSet = { { choiceID = 1, menuName = "Choice", vrCommands = { "VrChoice" }}}})
+  local on_hmi_call = EXPECT_HMICALL("VR.AddCommand", { cmdID = 1,
+          appID = default_app,
+          type = "Choice",
+          vrCommands = {"VrChoice"}})
+  on_hmi_call:Do(function(_,data)
+    grammarIDValue = data.params.grammarID
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+  EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+    self.currentHashID = data.payload.hashID
+  end)
+end
+
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Transport unexpected disconnect. Media app resume at FULL level")
```

Comment:
@istoimenova I updated all scripts according to your comments in bf8e134638d686a2778f4fffe3eb31c2a5c8bd21
please review